src/main/java/pl/touk/configuration/VipCustomerInitialValues.java

```diff
@@ -0,0 +1,29 @@
+package pl.touk.configuration;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import pl.touk.model.VipCustomer;
+import pl.touk.repository.ParkingRateRepository;
+
+import javax.annotation.PostConstruct;
+import java.util.Arrays;
+
+@Slf4j
+@Component("vipCustomerInitialValues")
+@RequiredArgsConstructor
+public class VipCustomerInitialValues {
+
+    private final ParkingRateRepository parkingRateRepository;
+
+    @PostConstruct
+    public void init() {
```

Comment:
czy ten bean nie powinien być uzależniony od środowiska?

Reply:
Moja aplikacja jest bardzo prosta. Ponad to mam baze danych w pamięci, więc nie rozróżniam tutaj środowisk. W aplikacji produkcyjnej owszem należałoby rozróżnić.